sitewit/services.py

```diff
@@ -335,6 +362,30 @@ def cancel_campaign_subscription(self, account_token, campaign_id,
             'api/subscription/cancel/campaign/search/', data=data,
             headers=self._get_account_auth_header(account_token)).json()
 
+    def cancel_display_campaign_subscription(self, account_token, campaign_id,
+                                             immediate=True):
+        """Cancel Display campaign subscription.
+
+        Cancel Display campaign subscription.
+
+        Args:
+            account_token (str): account token.
+            campaign_id (str): campaign to subscribe.
+            immediate (boolean): cancel immediately or wait till the billing
+            period ends.
+
+        Returns:
+            Please see response format here:
+            https://sandboxpapi.sitewit.com/Help/Api/
+            DELETE-api-subscription-cancel-campaign-display
+        """
+        data = {'campaignId': campaign_id,
+                'cancelType': 'Immediate' if immediate else 'EndOfCycle'}
+
+        return self.delete(
+            'api/subscription/cancel/campaign/display/', data=data,
```

Comment:
We're still passing the data as a form data.  We should probably switch to JSON here as well. (and above).